css/mobile-first.css

```diff
@@ -10,13 +10,30 @@ https://coolors.co/96bbbb-faf3dd-c8d5b9-f2e3bc-696d7d
 
 /* Global styles
 ------------------------------*/
-html, body {
-    overflow-x: hidden; /* removed unwanted white space on right side of mobile view */
-}
 * {
   box-sizing: border-box;
 }
 
+h1 {
+    font-size: 60px;
+    line-height: 1;
+    font-weight: bold;
```

Comment:
No big deal, but it's a good habit to start building now to organize your CSS properties. My personal preference (so you might develop your own), is like so:

1. Display, position
2. Geometry: width/height, top/bottom/left/right
3. Outer box: margin, border
4. Inner box: padding
5. Text: font size/weight, line-height, text-align, color
6. Other: opacity, transition, z-index

So for example:
```
div {
  position: relative,
  display: inline-block;
  left: 10px;
  top: 5px;
  margin-bottom: 10px;
  border-radius: 100%;
  padding: 2px;
  text-align: center;
  line-height: 1.2;
  font-size: 14px;
  color: #ff0000;
  opacity: 1;
  z-index: 100;
  transition: opacity 800ms;
}
```

Reply:
For me, this makes it so much easier when I come back to my code in a few weeks to make changes